host/src/main/java/com/instana/dc/host/AbstractHostDc.java

```diff
@@ -76,11 +95,45 @@ public void initMeters(OpenTelemetry openTelemetry) {
         initMeter(openTelemetry, HostDcUtil.MeterName.FILESYSTEM);
         initMeter(openTelemetry, HostDcUtil.MeterName.PROCESSES);
         initMeter(openTelemetry, HostDcUtil.MeterName.PAGING);
+        initMeter(openTelemetry, HostDcUtil.MeterName.IBMQMGR);
```

Comment:
This line should not be in the AbstraceHostDc class because other host DC implementation does not need it.

. We can override the initMeters method in MqApplianceDc class. 

Note: the new generic otel-host sensor does not care the scope name now.